assets/js/components/setup/site-verification.js

```diff
@@ -146,18 +101,6 @@ class SiteVerification extends Component {
 		} )();
 	}
 
-	/**
-	 * Insert siteURL to the option through the API
-	 * @param { string } siteURL
-	 */
-	async insertPropertyToSearchConsole( siteURL ) {
-		try {
-			data.set( 'modules', 'search-console', 'insert', { siteURL } );
-		} catch ( err ) {
-			throw err;
-		}
-	}
-
 	async insertSiteVerification( siteURL ) {
 		try {
 			const { shouldSetup } = this.props;
```

Comment:
The check for `shouldSetup` can be removed from here. It is already checked early in the component's lifecycle.

The `try catch` block can also be removed, similar to the cases mentioned above by @aaemnnosttv.